packages/lib-user/src/components/GroupStats/GroupStats.js

```diff
@@ -1,6 +1,6 @@
 'use client'
 
-import { string } from 'prop-types'
+import PropTypes from 'prop-types'
```

Comment:
This change is related to getting the `test` script (mocha test runner) to output as expected. See [this related comment](https://github.com/zooniverse/front-end-monorepo/pull/5475/files#r1369287374).